bin/cd.php

```diff
@@ -9,13 +9,14 @@
 use CentralDesktop\API\Workspaces\WorkspacesCommand;
 use Symfony\Component\Console\Application;
 use CentralDesktop\API\Auth\AuthCommand;
+use CentralDesktop\API\Pagination\PaginationCommand;
 
 $application = new Application();
 $application->add(new AuthCommand());
 $application->add(new WorkspacesCommand());
 $application->add(new UsersCommand());
 $application->add(new UploadFileCommand());
-
+$application->add(new PaginationCommand());
```

Comment:
Is this really a top level function or should it be a set of args added to all commands?